libs/vscode-ui/components/src/lib/autocomplete/autocomplete.component.ts

```diff
@@ -1,16 +1,222 @@
-import { Component, Input, ChangeDetectionStrategy } from '@angular/core';
+import {
+  Component,
+  OnDestroy,
+  Input,
+  SimpleChanges,
+  OnChanges,
+  ViewChild,
+  ElementRef,
+  OnInit,
+  ViewEncapsulation
+} from '@angular/core';
+import {
+  ControlValueAccessor,
+  NG_VALUE_ACCESSOR,
+  FormControl
+} from '@angular/forms';
+import {
+  map,
+  tap,
+  takeUntil,
+  switchMap,
+  startWith,
+  scan,
+  debounceTime,
+  filter,
+  shareReplay
+} from 'rxjs/operators';
+import {
+  Subject,
+  BehaviorSubject,
+  Observable,
+  fromEvent,
+  merge,
+  of
+} from 'rxjs';
 import { Schema } from '@angular-console/schema';
 
-/**
- * Not built yet - using a datalist temporarily to loosely mimic functionality
- */
+export enum AutocompleteNavKeys {
+  Enter = 'Enter',
+  ArrowUp = 'ArrowUp',
+  ArrowDown = 'ArrowDown'
+}
+
+export const AUTOCOMPLETE_NAV_KEYS = Object.values(AutocompleteNavKeys);
```

Comment:
make this into an es6 Set

expost const AUTOCOMPLETE_NAV_KEYS = new Set('Enter', 'ArrowUp', 'ArrowDown');

Reply:
Good call!